cmd/kvd/main.go

```diff
@@ -28,7 +29,11 @@ import (
 )
 
 // kvd custom flags
-const flagInvCheckPeriod = "inv-check-period"
+const (
+	flagInvCheckPeriod         = "inv-check-period"
+	configMempoolEnableAuth    = "mempool.enable_authentication"
```

Comment:
Not sure how easy this is, but I would include a default config option in app.toml when `kvd init` is called for the first time. So, if installing from empty, `app.toml` would have 

```
[mempool]
enable_authentication = false
authorized_addresses = []
```

Additional nit:  Mempool options could also be added as flags to the `kvd start` command. It seems like that is the general pattern used, where options can be specified is `app.toml` or passed as flags to `kvd start`. Not a huge deal if we don't follow that pattern, but thought I would mention it.

Reply:
Adding the stuff to `kvd init` is awkward. The config and flags in cosmos are a bit all over the place, and mostly of hard coded. However adding the additional flags is easy enough.
So I've added flags for the mempool options which at least give some visibility and documentation.